src/main/java/org/jenkinsci/plugins/docker/traceability/model/DockerTraceabilityReport.java

```diff
@@ -70,7 +71,7 @@ public DockerTraceabilityReport() {
     
     public DockerTraceabilityReport(@Nonnull Event event, @Nonnull Info hostInfo, 
             @CheckForNull InspectContainerResponse container, 
```

Comment:
I doubt, because we don't expect this situation if we track deployment events for containers.
@jtnord, WDYT?